### ISSUE TITLE
BUG | Missing linked names for Python commands

### ISSUE THREAD

**Issue by airvzxf** [shuru-project/shuru#28]
# BUG

## GENERAL

There are certain errors when running some commands that are installed with Python. This is because the installation does not create the link names, so we are forced to add the character “3” after each command, for example: pip3, or idle3.

## SOLUTION

A function was previously created to generate these links on Linux. The task is as simple as adding the missing commands, which are shown below.
- 2to3
- idle3
- pydoc3
- pip3

The source code is in the file `src/version_manager/python_version_manager.rs`:
```rust
fn link_names_python(…) -> Result<(), Error> {
    …
}
```

**Comment by harshdoesdev** [comment]
Thanks for raising this issue. Would you like to work on it, or should it be assigned to someone else?

**Comment by airvzxf** [comment]
> Thanks for raising this issue. Would you like to work on it, or should it be assigned to someone else?

No, I wouldn't like to work on it. I want to give other team members a chance to work on this bug. Especially since this is a beginner level fix. If no one is interested, I could do it.

**Comment by harshdoesdev** [comment]
Thanks, @airvzxf! @prxjwal, @Aditya-PS-05, would you guys be interested in picking it up? It’s a cool Rust-related issue.

**Comment by Aditya-PS-05** [comment]
Can you assign it to me?

**Comment by harshdoesdev** [comment]
Hi @Aditya-PS-05, I have assigned this issue to you. Please feel free to ping us on Discord or here if you have any questions or need further context. Thanks!

**Comment by airvzxf** [comment]
Hi, @Aditya-PS-05. I wish you are good.

Questions regarding this issue:
* Do you have some deadline for this issue?
* What is your advance?

**Comment by Aditya-PS-05** [comment]
I was busy with my college exams. Now, they are over. Now, I will finish it as soon as possible.
